### PR TITLE
ci: script changes to run minikube with none driver

### PR DESCRIPTION
**Pull request by Madhu-1**
install required crio to run minikube with none driver and added if check to exclude minikube arguments which doesnot required for none driver.
    
Signed-off-by: Madhu Rajanna <madhupr007@gmail.com>

### CODE REVIEW

**Comment by nixpanic** [comment]
I think you also need to modify https://github.com/ceph/ceph-csi/blob/ci/centos/single-node-k8s.sh

**Comment by Madhu-1** [comment]
> I think you also need to modify https://github.com/ceph/ceph-csi/blob/ci/centos/single-node-k8s.sh

Yes for now hardcoding the script to start cluster with none driver for testing, if everything goes well i move some installation and configurtion to above script

File: scripts/minikube.sh [hunk 1]
```diff
@@ -176,6 +177,14 @@ else
     DISK_CONFIG=""
 fi
 
+EXTRA_MINIKUBE_ARGS=${EXTRA_MINIKUBE_ARGS:-"--container-runtime=cri-o"}
+
+function install_crio() {
+    dnf install jq conntrack docker -y
```
Comment (nixpanic) [inline scripts/minikube.sh]: please do not use Docker, use Podman instead.

Isn't there a repository that provides CRIO RPMs?

**Comment by nixpanic** [comment]
@mergifyio rebase

**Comment by nixpanic** [comment]
#3419 and #3420 can hopefully replace this PR.

**Pull request closed without merging**